diff --git a/tools/ui/src/lib/components/app/actions/ActionIconCopyToClipboard.svelte b/tools/ui/src/lib/components/app/actions/ActionIconCopyToClipboard.svelte
index a21553267..1fa0e7b01 100644
--- a/tools/ui/src/lib/components/app/actions/ActionIconCopyToClipboard.svelte
+++ b/tools/ui/src/lib/components/app/actions/ActionIconCopyToClipboard.svelte
@@ -3,10 +3,12 @@
 	import { Copy } from '@lucide/svelte';
 	import { ICON_CLASS_DEFAULT } from '$lib/constants';
 	import { copyToClipboard } from '$lib/utils';
+	import { TooltipSide } from '$lib/enums';
 
 	export let ariaLabel: string = 'Копировать в буфер';
 	export let canCopy: boolean = true;
 	export let text: string;
+	export let tooltipSide: TooltipSide = TooltipSide.LEFT;
 </script>
 
 <ActionIcon
@@ -15,4 +17,5 @@
 	iconSize={ICON_CLASS_DEFAULT}
 	onclick={() => canCopy && copyToClipboard(text)}
 	tooltip={ariaLabel}
+	{tooltipSide}
 />
