diff --git a/tools/ui/src/lib/components/app/chat/ChatMessages/ChatMessageStatistics/ChatMessageStatistics.svelte b/tools/ui/src/lib/components/app/chat/ChatMessages/ChatMessageStatistics/ChatMessageStatistics.svelte
index 4f26671be..bd99a5098 100644
--- a/tools/ui/src/lib/components/app/chat/ChatMessages/ChatMessageStatistics/ChatMessageStatistics.svelte
+++ b/tools/ui/src/lib/components/app/chat/ChatMessages/ChatMessageStatistics/ChatMessageStatistics.svelte
@@ -258,7 +258,7 @@
 				class="bg-transparent"
 				icon={WholeWord}
 				tooltipLabel="Всего сгенерировано токенов"
-				value="{agenticTimings!.llm.predicted_n.toLocaleString()} tokens"
+				value="{agenticTimings!.llm.predicted_n.toLocaleString()} токенов"
 			/>
 
 			<ChatMessageStatisticsBadge
